pkg/service.go

```diff
@@ -0,0 +1,19 @@
+package service
+
+import (
+	"context"
+	"errors"
+
+	pl_health_service "github.com/Merteg/pl-health-service/proto"
+)
+
+type Server struct {
```

Comment:
Server is not a very nice name
When you name something in go, always think about it in context of the import - in your case it will be service.Server - sound not very nice isn't it?
service.Health or service.HealthRegistry would be much better as describes what kind of service do we provide